tutorials/notebooks/keras/ptq/example_keras_activation_threshold.ipynb

```diff
@@ -0,0 +1,866 @@
+      "metadata": {
+        "id": "eda9ad33-f88c-4178-8f19-bac6b2b2e97b"
+      },
+      "outputs": [],
+      "source": [
+        "REPRESENTATIVE_DATASET_FOLDER = './imagenet/val'\n",
+        "BATCH_SIZE = 20\n",
+        "fraction =0.001\n",
+        "representative_dataset_gen = tutorial_tools.get_representative_dataset(fraction, REPRESENTATIVE_DATASET_FOLDER, BATCH_SIZE)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "4a1e9ba6-2954-4506-ad5c-0da273701ba5",
+      "metadata": {
+        "id": "4a1e9ba6-2954-4506-ad5c-0da273701ba5"
+      },
+      "source": [
+        "## MCT quantisation"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "55edbb99-ab2f-4dde-aa74-4ddee61b2615",
+      "metadata": {
+        "id": "55edbb99-ab2f-4dde-aa74-4ddee61b2615"
+      },
+      "source": [
+        "This step we load the model and quantise with two methods of threshold error calculation: no clipping and MSE.\n",
+        "\n",
+        "No clipping chooses the lowest Power of two threshold that does not loose any data to its threshold.\n",
+        "\n",
+        "MSE chooses a Power of two threshold that results in the least difference between the float distribution and the quantised distribution.\n",
+        "\n",
+        "This means no clipping will often result in a larger threshold, which we will see later in this tutorial."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "First we load mobilenetv2 from the keras library"
+      ],
+      "metadata": {
+        "id": "VMrcPUN6jPlB"
+      },
+      "id": "VMrcPUN6jPlB"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "c431848f-a5f4-4737-a5c8-f046a8bca840",
+      "metadata": {
+        "id": "c431848f-a5f4-4737-a5c8-f046a8bca840"
+      },
+      "outputs": [],
+      "source": [
+        "from keras.applications.mobilenet_v2 import MobileNetV2\n",
+        "float_model = MobileNetV2()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Quantisation perameters are defined."
+      ],
+      "metadata": {
+        "id": "Pd8blHyKjWay"
+      },
+      "id": "Pd8blHyKjWay"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "ca971297-e00b-44b5-b9e1-e57ba5843e38",
+      "metadata": {
+        "id": "ca971297-e00b-44b5-b9e1-e57ba5843e38"
+      },
+      "outputs": [],
+      "source": [
+        "from model_compression_toolkit import QuantizationErrorMethod\n",
+        "\n",
+        "# Specify the IMX500-v1 target platform capability (TPC)\n",
+        "tpc = mct.get_target_platform_capabilities(\"tensorflow\", 'imx500', target_platform_version='v1')\n",
+        "\n",
+        "# Set the following quantization configurations:\n",
+        "# Choose the desired QuantizationErrorMethod for the quantization parameters search.\n",
+        "# Enable weights bias correction induced by quantization.\n",
```

Comment:
why do we need z threshold and shift negative in this tutorial? I think it is just confusing because its not mandatory for threshold selection)
Just use the default QuantizationConfig and override only the error method (because this is what we are trying to demonstrate in this tutorial)